libs/challenge-registry/home/src/_lib-theme.scss

```diff
@@ -0,0 +1,5 @@
+@use './lib/featured-challenge-list/featured-challenge-list-theme' as featured;
```

Comment:
Consider using a more descriptive name, e.g. `featuredChallengeList` or `featured-challenge-list`.

Reply:
has changed to `featured-challenge-list`